### DISCUSSION TITLE
企业版有docker镜像吗

### DISCUSSION THREAD

**Discussion by shijiew** [lsky-org/lsky-pro#691] [问答]
请问一下兰空图床企业版能否使用docker部署？

**Comment by WenkaiZhou** [comment]
企业版创建Docker构建方法 ([Docker-LskyProEnterpris](https://github.com/WenkaiZhou/Docker-LskyProEnterprise))，有需要的朋友可以交流下，面向小白 (自身就是小白)，大佬轻喷，有问题我就改～
